WcaOnRails/app/models/regulation.rb

```diff
@@ -21,6 +23,10 @@ def limit(number)
     first(number)
   end
 
+  def self.find(id)
```

Comment:
Sorry for my previous comment, I do agree with you that this makes more sense as `find_or_nil`.